### PR TITLE
[Docs] Add Chinese version of model_zoo.md

### PR DESCRIPTION
**Pull request by liufeinuaa**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add Chinese version of `docs/zh_cn/model_zoo.md`.

## Modification
Add Chinese version of `docs/zh_cn/model_zoo.md`.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/model_zoo.md [hunk 1]
```diff
@@ -16,9 +16,9 @@
 - [KFIoU](../../configs/kfiou/README.md) (arXiv)
 - [G-Rep](../../configs/g_reppoints/README.md) (stay tuned)
 
-### Results on DOTA v1.0
+### DOTA v1.0 数据集上的结果
 
-|    Backbone   |    mAP   | Angle | lr schd | Mem (GB) | Inf Time (fps) | Aug | Batch Size | Configs | Download |
+|    骨干网络   |    mAP   | 角度编码方式 | 训练策略 | 显存占用 (GB) | 推理时间 (fps) | 增强方法 | Batch Size | 配置 | 下载 |
```
Comment (yangxue0827) [inline docs/zh_cn/model_zoo.md:21]: Batch Size->批量大小

**Review by zytx121 — COMMENTED** [review]
LGTM.

**Review by zytx121 — APPROVED** [review]
LGTM.

File: docs/zh_cn/model_zoo.md (hunk 1 above)
Comment (liufeinuaa) [reply docs/zh_cn/model_zoo.md]: ok, im fix it

File: docs/zh_cn/model_zoo.md [hunk 2]
```diff
@@ -49,7 +49,7 @@
 | ResNet50 (1024,1024,200) | 79.66 | le90 | 1x |   | 13.7 | MS+RR | 2 | [roi_trans_r50_fpn_1x_dota_ms_le90](../../configs/roi_trans/roi_trans_r50_fpn_1x_dota_ms_le90.py) | [model](https://download.openmmlab.com/mmrotate/v0.1.0/roi_trans/roi_trans_r50_fpn_1x_dota_ms_rr_le90/roi_trans_r50_fpn_1x_dota_ms_rr_le90-fa99496f.pth) &#124; [log](https://download.openmmlab.com/mmrotate/v0.1.0/roi_trans/roi_trans_r50_fpn_1x_dota_ms_rr_le90/roi_trans_r50_fpn_1x_dota_ms_rr_le90_20220205_171729.log.json)
 | ReResNet50 (1024,1024,200) | 79.87 | le90 | 1x |  | 4.0 | MS+RR | 2 | [redet_re50_refpn_1x_dota_ms_rr_le90](../../configs/redet/redet_re50_refpn_1x_dota_ms_rr_le90.py) | [model](https://download.openmmlab.com/mmrotate/v0.1.0/redet/redet_re50_fpn_1x_dota_ms_rr_le90/redet_re50_fpn_1x_dota_ms_rr_le90-fc9217b5.pth) &#124; [log](https://download.openmmlab.com/mmrotate/v0.1.0/redet/redet_re50_fpn_1x_dota_ms_rr_le90/redet_re50_fpn_1x_dota_ms_rr_le90_20220206_105343.log.json)
 
-- `MS` means multiple scale image split.
-- `RR` means random rotation.
+- `MS` 表示多尺度图像分割增强。
```
Comment (ZwwWayne) [inline docs/zh_cn/model_zoo.md:52]: 多尺度图像增强

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**